### PR TITLE
media-libs/svt-av1: add 0.8.7

### PR DESCRIPTION
**Pull request by telans**

### CODE REVIEW

File: media-libs/svt-av1/svt-av1-0.8.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	SRC_URI="https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v${PV}/SVT-AV1-v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc -x86" # -x86: https://github.com/AOMediaCodec/SVT-AV1/issues/1231
+	S="${WORKDIR}/SVT-AV1-v${PV}"
+fi
+
+# Also see "Alliance for Open Media Patent License 1.0"
+LICENSE="BSD-2 Apache-2.0 BSD ISC LGPL-2.1+ MIT"
+SLOT="0"
+
+DEPEND="amd64? ( dev-lang/yasm )"
```
Comment (ionenwks) [inline media-libs/svt-av1/svt-av1-0.8.7.ebuild:24]: Shouldn't this be BDEPEND? I don't believe it's using libyasm.

Beside this nitpick, I see no issues here -- 9999 still builds too.

Comment (telans) [reply media-libs/svt-av1/svt-av1-0.8.7.ebuild]: fixed (finally)

File: media-libs/svt-av1/Manifest [hunk 2]
```diff
@@ -1 +1,2 @@
-DIST svt-av1-0.8.6.tar.gz 6416120 BLAKE2B c30d3bfc2dba6d7964625c4505bb135aa335adf195df91ad058fb02496276cabdc651b09854a2d14136352a434e08f4974148e03e7d54bde0924d25a1c4524cd SHA512 07e7353e35bc581b23c71913e65f5fd27566f4bf1d317d474d57d5bb2b82f380f713a00be19bd3f703935da50f81f3abf1d4ae38fbdf253385edf423737c0543
+DIST svt-av1-0.8.6.tar.gz 6416293 BLAKE2B 98a0c5547f63a9d04b5365a242d1a6e7b8f54449547309e53e9907d11d61c853f07ea1d8a5988ec1b67ef94bfd137ff1664f4352d9181cf96b5629f292c97f1c SHA512 535c073e53d99ed55efa0b7577a886dff136ba41f2d92e84ef0463b707136e156b6c4029e3ff7709be8ce54b0a7e7257af833b706b4282d7fd2636635df757d5
```
Comment (ionenwks) [inline media-libs/svt-av1/Manifest:1]: Changing Manifest for a file that's already mirror'ed needs to rename the tarball[1]. I usually use ${P}-r1 not to have a huge timestamp but that's fine too.

Also, can you squash commits a bit? S=, Manifest, and HOMEPAGE are all related changes.

[1] https://devmanual.gentoo.org/general-concepts/mirrors/index.html#replacing-automatically-mirrored-files

Comment (telans) [reply media-libs/svt-av1/Manifest]: Thanks for the link. How is this looking?

**Review by ionenwks — APPROVED** [review]

**Pull request closed without merging**